data/README.md

```diff
@@ -28,6 +28,14 @@ Gene stable ID, Transcript stable Id, HGNC Symbol, HGNC ID
 5. Click on `Results`, and export all results to a `TSV` file.
 6. Copy over the downoaded file to replace [ensembl_biomart_geneids_grch37.p13.txt](ensembl_biomart_geneids_grch37.p13.is_canonical.txt).
 
+##### Exon information in Ensembl endpoint
+1. Go to Ensembl FTP site ([ftp://ftp.ensembl.org/pub/grch37/](ftp://ftp.ensembl.org/pub/grch37/))
+2. Click desired Ensembl Release
+3. Click `gff3`
+4. Click `homo_sapiens`
+5. Click Homo_sapiens.GRCh37.[release#].gff3.gz and download will start (currently the file from release 87 is used)
```

Comment:
why 87?

Reply:
Apparantly the GFF file is not update in every release. Therefore, in the latest release [ftp Ensembl release 91](ftp://ftp.ensembl.org/pub/grch37/release-91/gff3/homo_sapiens/) the GFF file is still from release 87.